svelte.config.js

```diff
@@ -12,7 +12,9 @@ const config = {
 		// If your environment is not supported, or you settled on a specific environment, switch out the adapter.
 		// See https://svelte.dev/docs/kit/adapters for more information about adapters.
 		adapter: adapter()
-	}
+	},
+	edge: false,
+	split: false
```

Comment:
took this from the [docs](https://svelte.dev/docs/kit/adapter-netlify):

```
import adapter from '@sveltejs/adapter-netlify';

export default {
	kit: {
		// default options are shown
		adapter: adapter({
			// if true, will create a Netlify Edge Function rather
			// than using standard Node-based functions
			edge: false,

			// if true, will split your app into multiple functions
			// instead of creating a single one for the entire app.
			// if `edge` is true, this option cannot be used
			split: false
		})
	}
};
```

Reply:
I don't recall the default values off the top of my head, but I figured it wouldn't hurt to be explicit